packages/fxa-js-client/tests/mocks/request.js

```diff
@@ -223,6 +227,21 @@ define(['client/lib/errors', 'tests/lib/push-constants'], function(
       status: 200,
       body: '{"subscriptions":[{"foo":"bar"}]}',
     },
+    securityEvents: {
```

Comment:
@vbudhram see this, IMHO we can't call again securityEvents route after deleting securityEvents because this mock request will return this `securityEvents` object.

Reply:
@farhan787 you can create a new mock result and use that. Ex. `securityEventsEmptyResponse`

Reply:
Ok gotcha !